bumpversion/vcs.py

```diff
@@ -51,8 +51,8 @@ def is_usable(cls):
                 == 0
             )
         except OSError as e:
-            if e.errno == 2:
-                # mercurial is not installed then, ok.
+            if e.errno in (2, 13):
```

Comment:
Could you `import errno` and then use constants?
```suggestion
            if e.errno in (errno.ENOENT, errno.EACCES):
```

Reply:
No problem.   Change made, running tox locally and then I'll push.

Reply:
Changes pushed & ready for review.